src/Microsoft.AspNetCore.BuildTools.Tasks/Microsoft.AspNetCore.BuildTools.Tasks.nuspec

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yes, but it's gnarly. It's easier to just use nuspec than to tricking 'dotnet pack' into giving us the layout we want. I tried both.